bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/Display.java

```diff
@@ -1700,6 +1701,30 @@ Point getCursorLocationInPixels () {
 	return new Point (pt.x, pt.y);
 }
 
+Point getLocationInPixelsInDisplayCoordinateSystem(Point point) {
```

Comment:
I am thinking about naming of these methods. As the coordinate conversion methods are named map() below, I would probably propose to adapt the naming here as well to something like **map**LocationInPixels**To**DisplayCoordinateSystem

Reply:
Done